docs/src/FAQ.md

```diff
@@ -76,3 +76,52 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
 This means your package can depend on the light-weight ChainRulesCore.jl, and make ChainRulesTestUtils.jl a test-only dependency.
 
 Remember to read the section on [On writing good `rrule` / `frule` methods](@ref).
+
+## How do chain rules work for complex functions?
+
+`ChainRules.jl` follows the convention that `frule` applied to a function ``f(x + i y) = u(x,y) + i \, v(x,y)`` with perturbation ``\dot x + i \dot y`` returns the value and 
+```math
+\tfrac{\partial u}{\partial x} \, \dot x + \tfrac{\partial u}{\partial y} \, \dot y + i \, \Bigl( \tfrac{\partial v}{\partial x} \, \dot x + \tfrac{\partial v}{\partial y} \, \dot y \Bigr)
+,
+```
+and similarly `rrule` applied to the same function evaluates the function and returns a pullback which, when applied to adjoint ``\bar u + i \bar v``, returns
```

Comment:
I think `\bar u` is a bit confusing notation here, because it's usually used for conjugation. What do you think of `\mathrm{d}u` instead, since you are basically pulling back differential one-forms? The notation should probably also be explained briefly.

Reply:
Good point! I replaced both `\bar` and `\dot` with `\Delta`. This notation (and the earlier `dot` and  `bar` notation) are documented further up on this FAQ page.

Reply:
I don't think it makes sense to use `\Delta` for both, `frule` and `rrule`, because one lives in the tangent plane, whereas the other one lives in the cotangent plane.

Reply:
It's the notation proposed by the `ChainRules` documentation. Waiting for input from others.